test/bdd/fixtures/sidetree-mock/docker-compose.yml

```diff
@@ -16,7 +16,8 @@ services:
       - SIDETREE_MOCK_HOST=0.0.0.0
       - SIDETREE_MOCK_PORT=48326
       - SIDETREE_MOCK_API_TOKEN=rw_token
-      - SIDETREE_MOCK_DID_NAMESPACE=did:trustbloc:testnet.trustbloc.local
+      - SIDETREE_MOCK_DID_NAMESPACE=did:trustbloc
```

Comment:
This should have a hash of the genesis block/consortium config.

did:trustbloc:<consortium genesis hash>

@fqutishat @sandrask @bstasyszyn @Moopli

Reply:
Another topic for the hash is possible truncation.

Reply:
When configuring did-method we should have hash there. In sidetree-mock project we have did:trustbloc because of  interop tests.

Reply:
@fqutishat we will need a BDD test that shows the correct setup, including hash.

Reply:
@troyronda still WIP, I will add bddtest

Reply:
@troyronda @Moopli  is this consortium config https://github.com/trustbloc/trustbloc-did-method/blob/master/test/bdd/fixtures/wellknown/config.json

Reply:
Yes it is

Reply:
I added the hash